src/groonga_benchmark.c

```diff
@@ -2756,12 +2760,6 @@ usage(void)
   exit(1);
 }
 
-enum {
-  mode_default = 0,
-  mode_list,
-  mode_usage,
-};
```

Comment:
They are modes for command line options.
So we can't share them with `#define MODE_*`.